### PR TITLE
Remove supervisord.conf volume from Dockerfile

### PR DESCRIPTION
**Pull request by boehlke**
Not needed any more.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
We have to wait until the changes of new multiinstance backend is merged und in production. Otherwise current instances does not work with updated Dockerfile.

**Comment by boehlke** [comment]
@emanuelschuetze Bitte zusammen mit dem PR in multi-backend mergen. Ich teste das dann morgen/übermorgen auf dem Multi-Test-Server.

File: openslides/core/management/commands/collectstatic.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def handle(self, **options: Any) -> str:
 
     def collect(self) -> Dict[str, Any]:
         try:
-            destination_dir = os.path.join(settings.STATICFILES_DIRS[0], 'js')
+            destination_dir = os.path.join(settings.STATIC_ROOT, 'js')
```
Comment (emanuelschuetze) [inline openslides/core/management/commands/collectstatic.py]: @FinnStutzenstein Can you check if this is the best solution for OpenSlides core?

Comment (FinnStutzenstein) [reply openslides/core/management/commands/collectstatic.py]: @normanjaeckel May have a look

Comment (normanjaeckel) [reply openslides/core/management/commands/collectstatic.py]: I do not understand this solution. The command should collect the statics into the collected static dir and not into anything else.

Comment (boehlke) [reply openslides/core/management/commands/collectstatic.py]: @normanjaeckel In my estimation, STATICFILES_DIRS[0] is the first directory where static files should be collected from. STATIC_ROOT is the directory to collect files into! See https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-STATIC_ROOT and https://docs.djangoproject.com/en/2.0/ref/settings/#staticfiles-dirs ... do you come to the same conclusion?

Comment (normanjaeckel) [reply openslides/core/management/commands/collectstatic.py]: Hm. You are right. I mixed it up.

But the problem is, now the extra flags of the collectstatic command won't work properly (like --dry-run, --clear, --link, --ignore) and the STATIC_ROOT dir will get some stuff even if the user doesn't want this.

Comment (boehlke) [reply openslides/core/management/commands/collectstatic.py]: The problem is probably that the whole server/client code mix-up is a hack :-D
Do you need the features dry-run, clear etc?

Comment (emanuelschuetze) [reply openslides/core/management/commands/collectstatic.py]: @normanjaeckel Any concerns here? We need a solution that we can merge it soon for using with new multi-backend.

Comment (normanjaeckel) [reply openslides/core/management/commands/collectstatic.py]: I wrote a last question to @boehlke via email to fully understand the problem.

**Comment by normanjaeckel** [comment]
See #3804

**Pull request closed without merging**